### ISSUE TITLE
Story suggestions for ipywidgets

### ISSUE THREAD

**Issue by willingc** [jupyter/newsletter#63]
### Issue category

-  [x] Technical story suggestion (Projects, Releases, Code, Notebooks, etc.)

### Topic:

ipyvolume uses ipywidgets to create interactive documentation

### Headline:

*ipyvolume's documentation gets interactive with ipywidgets*

### Summary of story:

Maarten Breddels and the ipyvolume project is creating engaging and interactive
user documentation using Project Jupyter's ipywidgets. IPyvolume is a Python library 
to visualize 3d volumes and glyphs (e.g. 3d scatter plots), in the Jupyter notebook, 
with minimal configuration and effort.

### Link(s) to additional information:

- [ipyvolume documentation](https://ipyvolume.readthedocs.io.com)

### Links to images or photos to accompany the story:

- [ipyvolume documentation](https://ipyvolume.readthedocs.io.com)

**Comment by Ruv7** [comment]
@willingc  - I will include this in the list of possible topics for the April edition. There was a large amount of content of the March iteration. I will assume this is still a good topic unless you let me know it doesn't make sense to include...

**Comment by willingc** [comment]
Yes! Sylvain and Maarten Breddels would be good for any questions or more detail.

**Comment by Ruv7** [comment]
For the link provided above I'm getting an error message. Can you confirm this is the correct address @willingc @SylvainCorlay - https://ipyvolume.readthedocs.io.com/

**Comment by willingc** [comment]
https://ipyvolume.readthedocs.io Sorry about the bad link before.

**Comment by jasongrout** [comment]
You can add that Maarten is now active in the base ipywidgets development as well, contributing four merged PRs and one PR that is still open. (Besides reviewing other PRs)

**Comment by willingc** [comment]
For Ana, "We're pleased that Maarten is an active community contributor to the ipywidgets project as well. He recently contributed four merged PRs to ipywidgets. Thanks Maarten!"

**Comment by SylvainCorlay** [comment]
It could be worth mentionning the VAEX project (a data decimation engine) by the same author which motivated this new work on the front-end.

**Comment by Ruv7** [comment]
@SylvainCorlay - Here is the draft of the submission we are considering - if you want to make changes simply insert them and repost here. 

***

ipyvolume uses ipywidgets to create interactive documentation

Maarten Breddels and the ipyvolume project is creating engaging and interactive user documentation using Project Jupyter’s ipywidgets. IPyvolume is a Python library to visualize 3d volumes and glyphs (e.g. 3d scatter plots), in the Jupyter notebook, with minimal configuration and effort. We’re pleased that Maarten is an active community contributor to the ipywidgets project as well. He recently contributed four merged PRs to ipywidgets.

**Comment by SylvainCorlay** [comment]
How about:

Besides his work on ipyvolume, Maarten recently contributed four merged PRs to ipywidgets. We are pleased that he has become an active community contributor.

**Comment by Ruv7** [comment]
I like that edit, thanks. If you want to add anything about the VAEX project please feel free to do so.

**Comment by SylvainCorlay** [comment]
Thanks. Here is another proposal including VAEX.

[...] configuration and effort. It also serves as a front-end to the VAEX data decimation engine. Besides his work on ipyvolume and VAEX, Maarten recently contributed four merged PRs to ipywidgets. We are pleased that he has become an active community contributor.
